### PR TITLE
Update from 4.5.3 to 4.6.0

### LINKED ISSUE #231: Update to bootstrap 4.6.0
https://github.com/twbs/bootstrap/releases/tag/v4.6.0

### PR DESCRIPTION
**Pull request by cpsievert**
Closes #231. For reference, here are the changes in Bootstrap itself https://github.com/twbs/bootstrap/compare/v4.5.3...v4.6.0

### CODE REVIEW

File: tools/patches/006-bootswatch-slate.patch [hunk 1]
```diff
@@ -337,24 +337,24 @@ index 8a98f05..78b141a 100644
...
+ 
  .jumbotron {
 -  border: 1px solid rgba(0, 0, 0, .6);
 +  border: 1px solid rgba($contrast-fg, .6);
  }
-
+ 
  .list-group {
-   &-item:hover {
+   &-item-action:hover {
```
Comment (schloerke) [inline tools/patches/006-bootswatch-slate.patch:357]: This is new in `4.6.0`?

File: tools/patches/011-component-color-contrast.patch [hunk 2]
```diff
@@ -117,13 +117,13 @@ index 38aa9ae..fff34b9 100644
  $nav-divider-margin-y:              $spacer / 2 !default;
 @@ -775,8 +778,8 @@ $dropdown-link-color:               $gray-900 !default;
  $dropdown-link-hover-color:         darken($gray-900, 5%) !default;
- $dropdown-link-hover-bg:            $gray-100 !default;
+ $dropdown-link-hover-bg:            $gray-200 !default;
```
Comment (schloerke) [inline tools/patches/011-component-color-contrast.patch:120]: https://github.com/twbs/bootstrap/compare/v4.5.3...v4.6.0#diff-108a57a50d5ebe203f447ef7bf1e5e53826bdb82baab2baa2b41b4158cdd4d09R777

File: tools/patches/011-component-color-contrast.patch [hunk 3]
```diff
@@ -117,13 +117,13 @@ index 38aa9ae..fff34b9 100644
...
 
 -$dropdown-link-active-color:        $component-active-color !default;
  $dropdown-link-active-bg:           $component-active-bg !default;
 +$dropdown-link-active-color:        color-contrast($dropdown-link-active-bg, $component-active-color) !default;
 
- $dropdown-link-disabled-color:      $gray-600 !default;
+ $dropdown-link-disabled-color:      $gray-500 !default;
```
Comment (schloerke) [inline tools/patches/011-component-color-contrast.patch:126]: https://github.com/twbs/bootstrap/compare/v4.5.3...v4.6.0#diff-108a57a50d5ebe203f447ef7bf1e5e53826bdb82baab2baa2b41b4158cdd4d09R782

File: tools/patches/006-bootswatch-slate.patch (hunk 1 above)
Comment (schloerke) [reply tools/patches/006-bootswatch-slate.patch:357]: https://github.com/thomaspark/bootswatch/compare/v4.5.3...v4.6.0#diff-4fd0a22ae58c02bd0d4ad4ed0e7de80d4dd8e0a54235cc87e4ce0e2bedb06846R528

File: tools/patches/018-bs4-sectioned-comments.patch [hunk 4]
```diff
@@ -56,8 +56,8 @@ index fff34b9..9405ce7 100644
  
  $pagination-padding-y:              .5rem !default;
  $pagination-padding-x:              .75rem !default;
-@@ -822,6 +834,8 @@ $pagination-disabled-border-color:  $gray-300 !default;
- 
+@@ -843,6 +843,8 @@ $pagination-border-radius-sm:       $border-radius-sm !default;
+ $pagination-border-radius-lg:       $border-radius-lg !default;
```
Comment (schloerke) [inline tools/patches/018-bs4-sectioned-comments.patch:60]: New variables
https://github.com/twbs/bootstrap/compare/v4.5.3...v4.6.0#diff-108a57a50d5ebe203f447ef7bf1e5e53826bdb82baab2baa2b41b4158cdd4d09R821-R822

File: tools/patches/021-add-navbar-bg-vars.patch [hunk 5]
```diff
@@ -1,10 +1,10 @@
...
-index 7a42c44..aee26ce 100644
+index af39c76..e332bbc 100644
 --- a/inst/lib/bs/scss/_variables.scss
 +++ b/inst/lib/bs/scss/_variables.scss
-@@ -747,6 +747,9 @@ $navbar-toggler-padding-x:          .75rem !default;
- $navbar-toggler-font-size:          $font-size-lg !default;
- $navbar-toggler-border-radius:      $btn-border-radius !default;
+@@ -749,6 +749,9 @@ $navbar-toggler-border-radius:      $btn-border-radius !default;
+ 
+ $navbar-nav-scroll-max-height:      75vh !default;
```
Comment (schloerke) [inline tools/patches/021-add-navbar-bg-vars.patch:7]: Added in https://github.com/twbs/bootstrap/compare/v4.5.3...v4.6.0#diff-108a57a50d5ebe203f447ef7bf1e5e53826bdb82baab2baa2b41b4158cdd4d09R734-R735

File: tools/patches/021-add-navbar-bg-vars.patch [hunk 6]
```diff
@@ -1,10 +1,10 @@
 diff --git a/inst/lib/bs/scss/_variables.scss b/inst/lib/bs/scss/_variables.scss
-index 7a42c44..aee26ce 100644
+index af39c76..e332bbc 100644
 --- a/inst/lib/bs/scss/_variables.scss
 +++ b/inst/lib/bs/scss/_variables.scss
-@@ -747,6 +747,9 @@ $navbar-toggler-padding-x:          .75rem !default;
- $navbar-toggler-font-size:          $font-size-lg !default;
```
Comment (schloerke) [inline tools/patches/021-add-navbar-bg-vars.patch:6]: `$navbar-toggler-font-size` isn't used anymore?

**Review by schloerke — APPROVED** [review]
Only question is on the removal of `$navbar-toggler-font-size`.

Otherwise LGTM

File: tools/patches/021-add-navbar-bg-vars.patch (hunk 6 above)
Comment (cpsievert) [reply tools/patches/021-add-navbar-bg-vars.patch:6]: It is https://github.com/rstudio/bslib/pull/254/files#diff-6d46872b8309f0929d61994a2ec3fc406b3dc05588acd0ff002ac58c74bbaabcR747

(It's just no longer included in the patch file)

**Pull request merged**